back/app/views/application/_header.mjml

```diff
@@ -1,5 +1,5 @@
 
-<% if preheader %>
+<% unless header_logo_only? %>
```

Comment:
Wondering whether it could be better to not make this conditional logic in the existing partials/template (here and in the other places), but rather give it its own layout, when it's only for this one campaign?

Think a separate layout could be cleaner in this case, avoiding the conditional and the extra parameter. The shared content seems to be already in partials.

Reply:
This is getting a bit messy indeed, will give it a light refactor to introduce its own layout